internal/exporter/prometheus/collector/cpuinfo.go

```diff
@@ -8,30 +8,8 @@ import (
 	"sync"
 
 	prom "github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/procfs"
 )
 
-// procFS is an interface for CPUInfo.
```

Comment:
Can this refactor /reorg be a separate commit?

Reply:
perhaps if this refactor was done, comment would ask for a refactor